sdks/v4-sdk/package.json

```diff
@@ -0,0 +1,96 @@
+{
+  "name": "@uniswap/v4-sdk",
+  "description": "⚒️ An SDK for building applications on top of Uniswap V4",
+  "repository": "https://github.com/Uniswap/sdks.git",
+  "keywords": [
+    "uniswap",
+    "ethereum"
+  ],
+  "license": "MIT",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "module": "dist/v4-sdk.esm.js",
+  "files": [
+    "dist"
+  ],
+  "engines": {
+    "node": ">=10"
```

Comment:
nit: we can probably support node version much later than 10 for v4-sdk. ur-sdk supports >= 14, so we can probably do that for v4-sdk as well to start.